recipe/meta.yaml

```diff
@@ -1,59 +1,84 @@
 # version numbering is explained at https://stackoverflow.com/questions/42730478/version-numbers-for-visual-studio-2017-boost-and-cmake
 #     This version number follows the "platform toolset"
-{% set vcver="14" %}
-{% set vsyear="2015" %}
-{% set fullver="14.0.25420" %}
+{% set vcver="14.1" %}
+{% set vsyear="2017" %}
+{% set fullver="14.16"%}
```

Comment:
Use `14.16.27012` here?

Reply:
yup! I just noticed that when looking at the CI logs